deps.toml

```diff
@@ -74,8 +74,8 @@ platform-testcontainers-postgresql = { module = "org.testcontainers:postgresql",
 log4j-over-slf4j = { module = "org.slf4j:log4j-over-slf4j", version.ref = "slf4j" }
 appender-log4j2 = { module = "com.therealvan:appender-log4j2", version = "3.6.0" }
 aws-java-sdk-s3 = { module = "com.amazonaws:aws-java-sdk-s3", version = "1.12.6" }
-google-cloud-storage = { module = "com.google.cloud:google-cloud-storage", version = "2.2.2" }
-s3 = { module = "software.amazon.awssdk:s3", version = "2.16.84" }
+google-cloud-storage = { module = "com.google.cloud:google-cloud-storage", version = "2.13.0" }
```

Comment:
Is it possible to instead add constraints on the transitive dependencies that these pull, same as you did above?

Reply:
Actually this sounds good to me...